erpnext/utilities/activation.py

```diff
@@ -59,11 +68,15 @@ def get_level():
 	):
 		activation_level += 1
 
-	level = {"activation_level": activation_level, "sales_data": sales_data}
+	level = {"activation_level": activation_level, "sales_data": sales_data, "records": records}
 
 	return level
 
 
+def get_new_records(doctype):
+	return frappe.db.count(doctype, dict(creation=("between", (last_24hrs, now()))))
```

Comment:
Since creation doesn't have index this will get quite slow. We can build timeseries info from total records too (total vs timestamp) 

So no need to capture this data separately.